packages/app/src/runs/RunTagCount.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <Tooltip
+    tab-index="0"
+    placement="bottom"
+  >
+    <RunTag
+      :aria-label="ariaLabel"
+      :title="`+${value}`"
+      :label="`+${value}`"
+      data-cy="runTagCount"
+    />
+    <template
+      v-if="tooltipData"
+      #popper
+    >
+      <ul
+        class="flex gap-[8px] flex-col max-w-full children:flex text-gray-300"
+        data-cy="runTagCount-tooltip"
+      >
+        <li
+          v-if="(tooltipData.flaky || 0 > 0)"
+          class="items-center overflow-hidden"
+        >
+          <div
+            class="flex border rounded flex-row gap-[8px] items-center h-6 border-white/20 text-orange-300 text-sm px-2 gap-x-1"
+          >
+            <span
+              class="font-medium pr-1 opacity-70"
+            >
+              {{ tooltipData.flaky }}
+            </span>
+            <div class="w-px my-[6px] h-6 border-white/20 border" />
+            <span class="font-semibold pl-1">
+              {{ t('specPage.flaky.badgeLabel') }}
+            </span>
+          </div>
+        </li>
+        <li
+          v-if="tooltipData.branchName"
+          class="items-center overflow-hidden"
+        >
+          <IconTechnologyBranchH
+            aria-hidden="true"
+            class="shrink-0 mr-2 icon-dark-gray-500"
+          />
+          <span
+            class="sr-only"
+          >
+            {{ t('runs.card.branchName') }}
+          </span>
+          <div class="grow-1 shrink-1 max-w-full overflow-hidden">
+            <div class="max-w-full break-words whitespace-normal">
+              {{ tooltipData.branchName }}
+            </div>
+            <div />
+          </div>
+        </li>
+        <li
+          v-if="tooltipData.tags && tooltipData.tags?.length > 0"
+          class="flex-col items-start max-w-full"
+        >
+          <span>Tags</span>
+          <ul
+            class="flex flex-col children:flex children:items-center w-full"
+          >
+            <li
+              v-for="tag in tooltipData.tags"
+              :key="tag"
+              class="relative overflow-hidden max-w-full"
+            >
+              <div
+                aria-hidden="true"
+                class=" shrink-0 pr-[8px]"
+              >
+                •
```

Comment:
```suggestion
                <IconDotOutlineSmall class="icon-dark-gray-500 icon-light-gray-500" />
```
Should be an icon instead of a bullet point.  Make sure to add the import for this as well.